test/integration/src/main/java/io/pravega/test/integration/demo/ClusterWrapper.java

```diff
@@ -122,30 +137,64 @@
 
     private ClusterWrapper() {}
 
-    public void initialize() {
+    @SneakyThrows
+    public void start() {
+        if (this.isAuthEnabled() && passwordAuthHandlerEntries == null) {
+            this.passwordAuthHandlerEntries = Arrays.asList(defaultAuthHandlerEntry());
+        }
+        if (this.controllerRestEnabled && this.controllerRestPort <= 0) {
+            this.controllerRestPort = TestUtils.getAvailableListenPort();
+        }
+        startZookeeper();
+        log.info("Started Zookeeper");
+        startSegmentStore();
+        log.info("Started Segment store");
+        startController();
+        log.info("Started Controller");
+    }
+
+    @Override
```

Comment:
Why did you move this method? I have another PR in flight touching this class and it's going to conflict with this one because of this change

Reply:
Since I was enhancing this class, I thought it'd be a good opportunity to move around methods so that public and private methods are collocated.